scss/components/_button.scss

```diff
@@ -44,7 +44,8 @@ $button-radius: $global-radius !default;
 
 /// Border width for hollow outline buttons
 /// @type Number
-$button-hollow-border-width: 1px !default;
+$button-border-width: 1px !default;
+$button-hollow-border-width: $button-border-width;
```

Comment:
Can we make this a `!default` as well so that if someone has an older settings file that manipulated this, it will still work?